backup-restore/recipes/getResources.sh

```diff
@@ -11,22 +47,30 @@ if [[ -n $2 ]]; then
 	fi
 
 	fusionNs=$(oc get spectrumfusion -A --no-headers | cut -d" " -f1)
-    brNs=$(oc -n "$fusionNs" get fusionserviceinstance ibm-backup-restore-service-instance -o json |jq -rc '[(.spec.parameters[]|"\n",.name,.value)]|@csv' | tr -d '"' | grep ",namespace," | cut -d"," -f3)
+        uname=$(uname)
+        case "$uname" in
+          "Darwin")
+             [[ "${job_name}" ]] && job_uid=$(oc get f$job_type $job_name -n $fusionNs -o json | jq '.metadata.uid' | tr -d '"') ;;
+          "Linux")
+             [ "${job_name+set}" ] && job_uid=$(oc get f$job_type $job_name -n $fusionNs -o json | jq '.metadata.uid' | tr -d '"') ;;
```

Comment:
[ "${job_name+set}" ] works on macOS as well; maybe we don't need the case statement here ?

Reply:
yes, thank you for catching this.